### PR TITLE
New package: py-statmorph

### PR DESCRIPTION
**Pull request by meyersbs**
- New package: py-statmorph
    - New dependency: py-photutils
    - Update dependency: py-astropy@5.1
        - New dependency: py-pyerfa

### CODE REVIEW

**Comment by meyersbs** [comment]
@spackbot fix style

File: var/spack/repos/builtin/packages/py-astropy/package.py [hunk 1]
```diff
@@ -72,7 +81,10 @@ def patch(self):
         # avoids issues with PyCode_New() in newer
         # versions of python in the distributed
         # cython-ized files
-        os.remove("astropy/cython_version.py")
+        if os.path.exists("astropy/cython_version.py"):
+            os.remove("astropy/cython_version.py")
+        else:
+            pass
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-astropy/package.py]: You can get rid of the else clause if it isn't used

File: var/spack/repos/builtin/packages/py-pyerfa/package.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    pypi = "pyerfa/pyerfa-2.0.0.1.tar.gz"
+
+    maintainers = ["meyersbs"]
+
+    version("2.0.0.1", sha256="2fd4637ffe2c1e6ede7482c13f583ba7c73119d78bef90175448ce506a0ede30")
+
+    # From setup.cfg
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-numpy@1.17:", type=("build", "run"))
+    depends_on("py-setuptools-scm", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pyerfa/package.py]:
```suggestion
    depends_on("py-setuptools-scm@3.4:+toml", type="build")
```

File: var/spack/repos/builtin/packages/py-statmorph/package.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    images.
+    """
+
+    homepage = "https://github.com/vrodgom/statmorph"
+    pypi = "statmorph/statmorph-0.4.0.tar.gz"
+
+    maintainers = ["meyersbs"]
+
+    version("0.4.0", sha256="7d1bb802baf3e203ac44c630a58c5049da5eb4d85091ac35e3f5c6ee4af8b05a")
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-statmorph/package.py:21]: Needs build dep on setuptools

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**